### PR TITLE
[RUBY-132] Integration tests and cucumber features for UNSET values

### PR DESCRIPTION
**Pull request by kishkaru**

### CODE REVIEW

File: integration/session_test.rb [hunk 1]
```diff
@@ -648,4 +648,60 @@ def test_raise_error_on_future_resolution_timeout
...
+  # @since 3.0.0
+  # @jira_ticket RUBY-132
+  # @expected_result UNSET values should be implicitly added to bind parameters, leaving not-set values unaffected.
+  #
+  # @test_category prepared_statements:binding
+  #
+  def test_unset_values
+    skip("UNSET values are only available in C* after 2.2") if CCM.cassandra_version < '2.2.0'
+
+    begin
```
Comment (stamhankar999) [inline integration/session_test.rb]: No need for this explicit begin (and the matching end).

**Comment by stamhankar999** [comment]
Looks good aside from the begin/end block thing. Fix that and you're good. :+1:

**Comment by avalanche123** [comment]
please rebase this onto 3.0.0b

**Comment by kishkaru** [comment]
Fixed mentioned issue, squashed and rebased.

**Comment by avalanche123** [comment]
looks good to me

**Pull request merged**